src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransaction.tsx

```diff
@@ -55,6 +56,8 @@ const GroupedTransaction: FC<GroupedTransactionProps> = ({
   const groupId = getGroupId(transactionGroup);
   const status = getGroupStatus(transactionGroup);
   const values = getGroupValues<TransactionType>(transactionGroup);
+  const hasZeroAddressTransaction =
+    getHasZeroAddressTransaction(transactionGroup);
```

Comment:
Still needed to preserve the fix done on https://github.com/JoinColony/colonyCDapp/pull/2945